kotlin-react-router/src/main/generated/react/router/components.kt

```diff
@@ -0,0 +1,18 @@
+@file:JsModule("react-router")
+@file:JsNonModule
+
```

Comment:
Redundant file?

Reply:
I am going to revise inputs/outputs in `karakum.config.json` files, I believe it will be fixed after this activity.